_posts/2023-03-16-enable-github-discussion.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: posts
+title: Enable GitHub Discussions for the Bazel repository
+authors:
+  - pcloudy
+---
+We are excited to announce that [GitHub Discussions](https://github.com/bazelbuild/bazel/discussions) is now enabled for the Bazel GitHub project. This is a new channel for the Bazel community to collaborate, share ideas, ask questions, and provide feedback on the Bazel project, all without the need to open issues.
```

Comment:
Should we differentiate when to use GitHub discussions and when to use Slack?

Reply:
don't have a list, that's a good idea to post it as a discussion!